golden/clr-angular.d.ts

```diff
@@ -232,7 +232,7 @@ export declare class ClrDatagrid<T = any> implements AfterContentInit, AfterView
     placeholder: ClrDatagridPlaceholder<T>;
     refresh: EventEmitter<ClrDatagridStateInterface<T>>;
     rowActionService: RowActionService;
-    /** @deprecated */ rowSelectionMode: boolean;
```

Comment:
Any idea why the @deprecated comments were removed? I guess the `ts-api-guardian` update must of dropped them?

Reply:
It appears so.